qiita_pet/support_files/doc/source/index.rst

```diff
@@ -3,34 +3,42 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to qiita's documentation!
+Welcome to Qiita's documentation!
 =================================
 
 Qiita (canonically pronounced cheetah) is a software package intended for
 analysis and administration of multi-omics datasets.
 
-Qiita provides a free and open platform for users to: Easily share and reuse
-existing data-sets in the form of studies. Perform analyses (by combining one
-or more studies/data-sets) using published and unpublished data that meet the
-standards as described by this documentation.  Easily interface with the EBI
-repository for automated deposition. Query and interact with Qiita data
-programmatically.
+Qiita provides a free and open platform for users to:
+1) Easily share and reuse existing data-sets in the form of studies.
+2) Perform analyses (by combining one or more studies/data-sets) using published and unpublished data that meet the
+standards as described by this documentation.
+3) Easily interface with the EBI repository for automated deposition. Query and interact with Qiita data programmatically.
 
-The latests tutorials can be found in:
-`CMI Qiita/GNPS workshop pages <http://cmi-workshop.readthedocs.io/en/latest/>`__.
+Looking for information about our new analysis pipeline? Please see the document here:
+  
+.. toctree::
+   :maxdepth: 1  
+   
+   analysisguide/index.rst
```

Comment:
the error is here: analysisguide/index as it should be: analysisguides/index.rst